src/main/java/org/folio/fqm/service/EntityTypeFlatteningService.java

```diff
@@ -39,7 +39,7 @@ public EntityType getFlattenedEntityType(UUID entityTypeId, String tenantId) {
   }
 
   private EntityType getFlattenedEntityType(UUID entityTypeId, EntityTypeSource sourceFromParent, String tenantId) {
-  // TODO: clean up
+    // TODO: clean up
```

Comment:
I cleaned it up :)

Reply:
Appreciate you taking the time to handle this massive piece of tech debt.